src/cache.rs

```diff
@@ -36,6 +35,7 @@ impl Cache {
         // $TLDR_RS_CACHE_DIR env variable.
```

Comment:
That was the next thing on my list, renaming everything from `tldr` to `tealdeer`.